doc/specs/#1571 - New Tab Menu Customization/#1571 - New Tab Menu Customization.md

```diff
@@ -97,6 +108,7 @@ There are five `type`s of objects in this menu:
     enabling all other profiles to also be accessible.
   - The "name" of these entries will simply be the name of the profile
   - The "icon" of these entries will simply be the profile's icon
+  - This won't include any profiles that have been included via `match` entries.

```

Comment:
What I was thinking is that it would be nice to have some combination of `matchProfile` and `remainingProfiles`: for example, you want to put your 2 most-used WSL profiles at the top, then a separator, and then "all remaining WSL profiles". Tricky to implement in code, but I think there is a good usecase for it.

Reply:
Interesting, maybe we want to just always treat "match" entries as basically "remainingProfiles that match that filter". I'm not sure how much use case we get for having multiple entries for the same profile in the menu. 

So we'd have to evaluate:
* all explicit `profile` entries
* then all `match` entries, using profiles not already specified
* then expand out `remainingProfiles` with anything not found above.

Reply:
Agreed!